packages/client/src/config.schema.json

```
@@ -179,6 +179,18 @@
                     "type": "number",
                     "default": 15000
                 },
+                "externalIp": {
+                    "anyOf": [
+                        {
+                            "type": "null"
+                        },
+                        {
+                            "type": "string",
+                            "format": "ipv4"
+                        }
+                    ],
+                    "default": null
+                },
```

You can simplify this to the following and then the value will be `undefined` by default. Allowing you to have type `string` instead of `ExternalIP = string | null`.

```suggestion
                "externalIp": {
                     "type": "string",
                     "format": "ipv4"
                },
```